### PR TITLE
Adding options for nmap and greppable output

### LINKED ISSUE #73: No output options
There does not appear to be a way to also create standard nmap output files while using this library. Is that something that could be considered for a feature or was it conscious omission?

### PR DESCRIPTION
**Pull request by brutalgg**
Seems like this was a pretty simple change so I just wrote the purposed options. 

Closes #73

### CODE REVIEW

**Review by elivlo — APPROVED** [review]
Thank you for the contribution @brutalgg!
Yep this was a simple task to implement but also a valuable improvement. 👍

LGTM @Ullaakut

File: nmap.go [hunk 1]
```diff
@@ -1556,3 +1556,19 @@ func WithUnprivileged() Option {
 		s.args = append(s.args, "--unprivileged")
 	}
 }
+
+// WithNmapOutput makes nmap output standard output to the filename specified.
```
Comment (elivlo) [inline nmap.go:1560]: This function will have no effect when using the `RunWithStreamer` function. I would suggest adding one more comment to the description.

**Review by elivlo — CHANGES_REQUESTED** [review]

File: nmap.go (hunk 1 above)
Comment (brutalgg) [reply nmap.go:1560]: @elivlo What are you seeing that isn't working? I executed basic_scan_streamer_interface and the functionality appears to be working normally.

Comment (elivlo) [reply nmap.go:1560]: Sure that was my bad. It is still working nomally.

**Review by elivlo — APPROVED** [review]

**Review by Ullaakut — APPROVED** [review]

**Pull request merged**